services/plugin/src/Plugin.tsx

```diff
@@ -179,15 +179,9 @@ export const Plugin = ({
                     console.error(err)
                 })
         }
-    }, [
-        memoizedPropsToPass,
-        inErrorState,
-        // The following should be pretty stable:
-        alertsAdd,
-        clientWidth,
-        height,
-        width,
-    ])
+        // Skip `width` and `height` props to avoid updates when changing size:
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [memoizedPropsToPass, inErrorState, alertsAdd, clientWidth])
```

Comment:
Instead of disabling the linter, what about having boolean state vars `hasHeight` and `hasWidth` and pass those instead to this effect?

Reply:
I think that would work, good suggestion 🙂

Reply:
or a ref instead of state?

Reply:
Since `height` and `width` are props, it would take some 'prev/current' ref juggling to do

The boolean prop works though :)